packages/ui/src/hooks/_utils/responsiveObserve.ts

```diff
@@ -0,0 +1,74 @@
+export type Breakpoint = 'mobileSmall' | 'mobileLarge' | 'tabletSmall' | 'tabletLarge' | 'desktopSmall' | 'desktopLarge'
+export type BreakpointMap = Record<Breakpoint, string>
+export type ScreenMap = Partial<Record<Breakpoint, boolean>>
+export type ScreenSizeMap = Partial<Record<Breakpoint, number>>
+
+export const responsiveArray: Breakpoint[] = ['mobileSmall', 'mobileLarge', 'tabletSmall', 'tabletLarge', 'desktopSmall', 'desktopLarge']
+
+export const responsiveMap: BreakpointMap = {
+    mobileSmall: '(max-width: 480px)',
```

Comment:
are you sure about naming conventions? Should we use upper case like condo const or camel case?

Reply:
used upper case here